withdraw-by-check/REPWRITERSPECS/BANNO.CHECK.WITHDRAW.V1.POW

```diff
@@ -535,11 +548,17 @@ PROCEDURE VALIDATEACCOUNT
 ** RETURN  VALIDATESTATUS          Validate status
 ** RETURN  SLAVAILABLEBAL          The available balance/loan advance
 ** RETURN  SLDESCRIPTION           The Share/Loan description
+** RETURN  INVACCTWARNSFOUND       Invalid Account Warnings Found List
+** RETURN  INVSLWARNSFOUND         Invalid Share/Loan Warnings Found List
+** RETURN  INVSLTYPEFOUND          The invalid Share/Loan Type if Found
 *]
 
  VALIDATESTATUS=VALIDATEACCTNOTFOUND
  SLAVAILABLEBAL=$0.00
  SLDESCRIPTION=""
+ INVACCTWARNSFOUND=""
```

Comment:
Maybe call these 3 xxxxLIST variables instead of xxxxFOUND??

Reply:
made suggested change